WordPress/Classes/ViewRelated/Jetpack/Branding/Button/JetpackButton.swift

```diff
@@ -68,8 +68,7 @@ class JetpackButton: UIButton {
     }
 
     private func configureButton() {
-        // TODO: Remove this when the modal presentation is added
-        isUserInteractionEnabled = false
+        isUserInteractionEnabled = FeatureFlag.jetpackPoweredBottomSheet.enabled
```

Comment:
This of course enables all buttons and interacting with other ones make them blink due to highlight state change. But it is feature flag already so we should be fine.

Reply:
Exactly: since we are releasing all the overlays at the same time, it's not an issue if we see them blinking and not yet producing anything while we temporarily enable the flag.